src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SscsCaseDataWrapperDeserializer.java

```diff
@@ -0,0 +1,543 @@
+package uk.gov.hmcts.reform.sscs.transform.deserialize;
```

Comment:
There is now a deserialiser in the common library which we should be using for callbacks instead of this